Sources/BoundedSynthesis/Options.swift

```diff
@@ -253,8 +253,8 @@ public struct BoSyOptions {
         do { // add warnings if spot options set but spot not seleced
             let args = self.spotOptions ?? self.spotSimplGoal.rawValue + " " + self.spotSimplLevel.rawValue
             try converter = initAutomatonConverter(autoTool, args: args)
-            if autoTool != "spot" && (args != "" || (spotSimplGoal == .any && spotSimplLevel == .small)){
-                print("Warning! Command line options for spot set but spot is not beeing used. Ignoring. \n Set options were \(args). \n")
+            if autoTool != "spot" && (spotSimplGoal != .any && spotSimplLevel != .small){
```

Comment:
And what happens if `spotOptions` is set?  
Also, this cannot detect whether the default values are passed.

A simple solution would be to add a boolean flag and set it to true while handling the options.  
This conditions would then become a check for `autoTool == "spot" && spot_option_set`.

Reply:
fixed condition in new commit. 

> And what happens if spotOptions is set?
Which should take precedence? The option string or the simplification-{goal,level} settings?